### PR TITLE
Adding Stochastic Variational Version

### PR DESCRIPTION
**Pull request by mbz**

### CODE REVIEW

**Comment by cbfinn** [comment]
Largely looks good to me, except that you should update the README appropriately, e.g. add a link to the paper, update the contributors, and document the appropriate flags to replicate results in the paper, etc.

**Comment by cbfinn** [comment]
@mbz Can you add an example flag usage to the README for the new flags?

**Comment by cbfinn** [comment]
Nevermind - I just saw it.

File: research/video_prediction/README.md [hunk 1]
```diff
@@ -87,12 +98,26 @@ details. If the `--use_state` option is not set, then the data only needs to
 contain image sequences, not states and actions.
 
 
+the stochastic version of the model exposes a handful of new arguments:
```
Comment (cbfinn) [inline research/video_prediction/README.md]: nitpic: Captitalize "The".

**Comment by cbfinn** [comment]
@tensorflow-jenkins can we have an update on this?

**Comment by zacwellmer** [comment]
Any updates on if this is going to be merged?

**Comment by mbz** [comment]
I consider this code deprecated. If you are looking for sv2p implementation
you can find (an slightly modified version of) it in Tensor2Tensor library:
https://github.com/tensorflow/tensor2tensor/blob/master/tensor2tensor/models/video/sv2p.py

**Pull request closed without merging**